### PR TITLE
Added category paginated display and job create form

### PR DESCRIPTION
**Pull request by vlad-bruma**

### CODE REVIEW

File: src/Controller/CategoryController.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        /** @var Pagerfanta $pagerFanta */
+        $pagerFanta = $this->getDoctrine()
+            ->getRepository(Job::class)
+            ->findActiveByCategoryPaginated($category, $this->getParameter('max_jobs_per_category'));
+
+        $pagerFanta->setCurrentPage($page);
+
+        return $this->render('category/show.html.twig', [
+            'category' => $category,
+            'paginatedJobs' => $pagerFanta
```
Comment (gregurco) [inline src/Controller/CategoryController.php]: add `,` at the last element in array

File: src/Form/JobType.php [hunk 2]
```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Category;
+use App\Entity\Job;
+
```
Comment (gregurco) [inline src/Form/JobType.php]: remove this new line

**Review by gregurco — CHANGES_REQUESTED** [review]

File: templates/job/create.html.twig [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1 class="text-center mt-3">Create new job</h1>
+
+    {{ form_start(form, {method: 'POST', action: url('job.create')}) }}
+        {{ form_widget(form) }}
+
+    <div class="form-group">
```
Comment (gregurco) [inline templates/job/create.html.twig]: add one more tab for this `div`. It should be on the same level as `form_widget`

**Comment by vlad-bruma** [comment]
Added adjustments

**Pull request merged**